pkg/pool-stable/test/StablePool.test.ts

```diff
@@ -529,11 +529,11 @@ describe('StablePool', function () {
           await multiExitGivenIn();
         });
 
-        it('does not revert if paused', async () => {
+        it('reverts if paused', async () => {
           await pool.pause();
 
           const bptIn = previousBptBalance.div(2);
-          await expect(pool.multiExitGivenIn({ from: lp, bptIn })).not.to.be.reverted;
+          await expect(pool.multiExitGivenIn({ from: lp, bptIn })).to.be.revertedWith('PAUSED');
```

Comment:
I'd remove all of the pause tests here and instead have tests on LegacyBasePool that check that exits and joins are disabled when paused.

Reply:
Sure, but we can call `_ensureNotPaused()` before calling `_onExitPoll(...)` in the BasePool.